cmk/gui/plugins/openapi/restful_objects/request_schemas.py

```diff
@@ -38,6 +38,7 @@
 MONITORED_HOST = fields.HostField(
     description="The hostname or IP address itself.",
     example="example.com",
+    should_exist=None,
```

Comment:
This makes me think that `should_exist` would be better of as a mandatory parameter, so that these kinds of errors would not be possible any more. Apparently the fact that `should_exist=True` is the default got forgotten or overlooked.